src/code/code_8012EC80.c

```diff
@@ -1,4 +1,5 @@
 #include "global.h"
+#include "interface/parameter_static/parameter_static.h"
 
 // Bit Flag array in which gBitFlags[n] is literally (1 << n)
 u32 gBitFlags[] = {
```

Comment:
Below this there a couple of `NegMasks` arrays (gEquipNegMasks and gUpgradeNegMasks) thoughts on using bitwise negation for them, i.e gEquipNegMasks  becomes `{~0x000F, ~0x00F0, ~0x0F00, ~0xF000}`

Reply:
I did notice that and tried it. For `u32 gUpgradeNegMasks`, yes it looks much better with the `~`. For `u16 gEquipNegMasks` though, it needs to be:

`{(u16)~0x000F, (u16)~0x00F0, (u16)~0x0F00, (u16)~0xF000}`

Which looks a little worse, so that's why I left it. However, it is done in other places in the repo so perhaps it's still worth doing that.

Reply:
Really it needs an explicit cast, even though the type is `u16`?

Reply:
What happens if you do it without the casts? NON-MATCHING, warning, or something else?

Reply:
![image](https://user-images.githubusercontent.com/47598039/147447888-d2a563ac-c01b-4693-8400-a5c9e3c997e4.png)
Right, now I remember. It matches just fine, I was misremembering. All that happens is that I get a warning for the last entry

Reply:
Only the last entry needs a cast to remove the warning though, so I think that's minor enough that it's fine to put it